### DISCUSSION TITLE
Terminal implementation

### DISCUSSION THREAD

**Discussion by albeph** [thebaselab/codeapp#723] [Ideas]
I would like to suggest some implementation for the terminal, like:

- keyboard shortcut for open terminal panel;
- possibility to open more than one terminal;
- option to close and reopen terminal via ssh, (ISSUE) because when I close it (with exit) there isn’t a way to reopen it;

Also:
- a way to remove some suggestion from the editor.
- A way to quick connect (via SFTP) and go to a specific folder, because is a little buggy, on the welcome page he suggest the folder on the SFTP server, but don’t ask me for face id, so he can’t connect to the folder, and once I connect to it, the welcome page disappear;

Thanks for reading these tips/bugs,
Have a nice day!

**Comment by hg0428** [comment]
Maybe a moveable tab system similar to replit would work.

**Comment by bummoblizard** [comment]
Thank you for the suggestions. The keyboard-shortcut for opening (closing) the terminal panel is Command-J.

Regarding the quick connect feature in the welcome page, it is indeed a bug. I will fix it in the next update.

**Reply by albeph** [reply]
I appreciate that you read my ideas!
Also it will be nice if I can tweak the all the keyboard shortcuts for a full customisable experience.
Great app, great job, keep pushing!
